pkg/completion/completion.go

```diff
@@ -24,3 +30,90 @@ func AutocompleteNone(cmd *cobra.Command, args []string, toComplete string) ([]s
 func AutocompleteDefault(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 	return nil, cobra.ShellCompDirectiveDefault
 }
+
+// AutocompleteCapabilities - Autocomplete linux capabilities options.
+// Used by --cap-add and --cap-drop.
+func AutocompleteCapabilities(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	// FIXME: get this list from containers/psgo/internal/capabilities/capabilities.go
+	LinuxCapabilities := []string{
```

Comment:
@rhatdan pointed out that the list in https://github.com/containers/psgo/blob/master/internal/capabilities/capabilities.go could be used. In order to use this:
1.  `psgo` has to export the capabilities list in some way.
2. `c/common` would need to import `psgo`

Reply:
@vrothberg WDYT?

Reply:
Actually we already have the list in pkg/capability/AllCapabilities() in containers/common now.  We should probably use this list in psgo and definitely here.

Reply:
Yes this is perfect. I updated the code.